app/scripts/views/marketing_snippet_ios.js

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Handles the marketing snippet on the 'ready' page.
+ *
+ * Shows `Get Sync on Firefox for Android` for users that complete
+ * signup for sync in Firefox Desktop.
+ */
+
+'use strict';
+
+define([
+  'views/base',
+  'lib/constants',
+  'stache!templates/marketing_snippet_ios'
+], function (BaseView, Constants, Template) {
+
+  var View = BaseView.extend({
```

Comment:
Let's extend this from MarketingSnippet (marketing_snippet.js) and reduce the duplicate code.